lib/uuidUtils.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const uuidRegEx = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/;
+const uuidV1Offset = 122192928000000000; //  100 nanosecond intervals since 15 October 1582
+
+const uuidUtils = {};
+
+/**
+ * Extract time in nanoseconds from UUID V1
+ *
+ * @param {string} uuidv1
+ * @return {Integer|null}
+ *
+ */
+uuidUtils.getTime = (uuidv1) => {
```

Comment:
This will return the number of nanoseconds from 1582-10-15 but time is expressed in milliseconds elapsed since 1970-01-01

Reply:
Added function to return ns and moved the ns to ms conversion to getTime()